db-tests/src/test/java/gov/nasa/jpl/aerie/database/UserSequenceTests.java

```diff
@@ -0,0 +1,209 @@
+package gov.nasa.jpl.aerie.database;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@SuppressWarnings("SqlSourceToSinkFlow")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class UserSequenceTests {
+  private final String sequenceDefinition = """
+        C BAKE_BREAD
+        C PEEL_BANANA \"fromStem\"
```

Comment:
Very Minor, ignorable: Because you're in a block string, you don't need to escape the `"` here!